runtime/src/lib.rs

```diff
@@ -483,8 +505,8 @@ impl_runtime_apis! {
 		fn duty_roster() -> parachain::DutyRoster {
 			Parachains::calculate_duty_roster()
 		}
-		fn active_parachains() -> Vec<parachain::Id> {
-			Parachains::active_parachains()
+		fn active_parachains() -> Vec<(parachain::Id, Option<parachain::CollatorId>)> {
```

Comment:
```suggestion
		fn active_parachains() -> Parachains {
```

as per comment https://github.com/paritytech/polkadot/pull/341/files#r308987015 we could reuse a type alias to simplify